src/frontend/apps/desk/.env.development

```diff
@@ -1,5 +1 @@
 NEXT_PUBLIC_API_URL=http://localhost:8071/api/v1.0/
```

Comment:
You can clean them here as well so:
https://github.com/numerique-gouv/people/blob/main/src/frontend/apps/desk/src/custom-next.d.ts#L23-L26

Reply:
done !